templates/parachain/runtime/src/configs/mod.rs

```diff
@@ -134,7 +133,7 @@ impl pallet_timestamp::Config for Runtime {
 	/// A timestamp: milliseconds since the unix epoch.
 	type Moment = u64;
 	type OnTimestampSet = Aura;
-	type MinimumPeriod = ConstU64<MINIMUM_PERIOD>;
+	type MinimumPeriod = ConstU64<0>;
```

Comment:
Ouf of curiosity - why this was set to 0?

Reply:
At first, it seemed I couldn't run the parachain runtime based chain spec with chopsticks because of an error which IIRC stated that: "Timestamp slot must match `CurrentSlot`. This likely means that the configured block time in the node and/or rest of the runtime is not compatible with Aura's `SlotDuration`". After reading the code for a bit I thought it might have to do with the `MinimumPeriod`, which I changed to `SLOT_DURATION / 2`, resulting in the newly generated chain spec to work with `chopsticks`.

For some reason rebuilding the runtime at a later day made things work without the `MinimumPeriod` change. I also tested this right now and things work as expected without the minimum period change.